readthedocs/projects/views/private.py

```diff
@@ -344,6 +346,31 @@ class ImportView(PrivateViewMixin, TemplateView):
     template_name = 'projects/project_import.html'
     wizard_class = ImportWizardView
 
+    def get(self, request, *args, **kwargs):
+        '''Display list of repositories to import
+
+        Adds a warning to the listing if any of the accounts connected for the
+        user are not supported accounts.
+        '''
+        deprecated_accounts = (
```

Comment:
Yeah, it requires manually unlinking them in the connections page. Or, rather, both can exist, we just need the Bitbucket OAuth2 provider. So as long as the user sets that up, everything will work -- albeit there will be a warning still. I considered making a view that unlinks the account and redirects to django-allauth, but felt that was over-complicating the issue.